src/pushprocessor.ts

```diff
@@ -96,6 +97,22 @@ const DEFAULT_OVERRIDE_RULES: IPushRule[] = [
             PushRuleActionName.DontNotify,
         ],
     },
+    {
+        // Make sure we don't notify on ACL changes - https://github.com/vector-im/element-web/issues/20788
+        rule_id: ".m.rule.server_acl",
```

Comment:
Where is this rule defined? I'm not seeing it in https://spec.matrix.org/v1.2/client-server-api/

Reply:
It isn't - we'll either go without it being there or wait for an MSC (which I will write soon, hopefully :sweat_smile:)

Reply:
We can't use `.m.rule` prefixes without it being specced

Reply:
I suppose a sensible way would be to make it `io.element.rule.server_acl` while we don't have an MSC?

Reply:
This alone won't be sufficient, because the server will be incrementing the notification_count so causing invalid unread counts without related notifications

Reply:
Ah, true :+1: , I'll write an MSC then, I suppose